client/src/features/users/Auth/ForgotPassword.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import SimpleSnackbar from '../../action/assets/SimpleSnackbar';
+
+import type { Theme } from '@material-ui/core/styles';
+
+import EmailInput from './EmailInput';
+
+const useStyles = makeStyles((theme: Theme) => {
+  const xPadding = 12;
+  const yPadding = 6;
+  const yMargin = 8;
+
+  return {
+    paper: {
+      maxWidth: 821 - theme.spacing(xPadding),
+      maxHeight: 732 - theme.spacing(yPadding),
+      borderRadius: 10,
+      marginTop: theme.spacing(yMargin),
+      marginBottom: theme.spacing(yMargin),
+      paddingTop: theme.spacing(yPadding),
+      paddingBottom: theme.spacing(yPadding),
+      paddingLeft: theme.spacing(xPadding),
+      paddingRight: theme.spacing(xPadding),
+      margin: 'auto',
+    },
+    header: { fontWeight: 'bold', marginBottom: 68 },
+    button: {
+      borderRadius: 0,
+      height: 62,
+      textTransform: 'none',
+    },
+  };
+});
+
+interface UserLoginData {
+  email: string;
+}
+
+const initialFormData: UserLoginData = {
+  email: '',
+};
+
+interface Error {
+  type: '' | 'email';
+  message: string;
+}
+
+function ForgotPassword() {
+  const classes = useStyles();
+  const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [error] = React.useState<Error | null>(null);
+  const [showSnackbar, setShowSnackbar] = React.useState<boolean>(false);
+  const [formData, setFormData] = React.useState(initialFormData);
+  const instructions =
+    "Enter your email and we'll send you a link to reset you password if you have an account.";
+  const emailResponse =
+    'If this email exists as a user you will be sent an email to reset your password.';
+
+  const generateForm = () => {
+    return (
+      <Grid container item xs={12}>
+        <form onSubmit={handleSubmit} style={{ width: '100%' }}>
+          <EmailInput
+            value={formData.email}
+            placeholder="jane@nonprofit.com"
+            onChange={handleChange}
+            error={error?.type === 'email' ? error.message : null}
```

Comment:
right now your error state is always null, because you don't have a `setError` ever being called (which would come from the `useState` line)
i say just hardcode pass in `null` for now, remove that error state from here, and write up an issue to move the error state management to live inside the EmailInput, not ideal to have to rewrite that everywhere

Reply:
makes sense. Thanks. Changed.